### PR TITLE
Add translation for new locale (gd)

### PR DESCRIPTION
**Pull request by gunchleoc**
## Description
New translation :)

I searched for "zh_tw" to find the configuration bits - hope I haven't missed anything!

## Related Issues
https://crowdin.com/project/whalebird/discussions/2

### CODE REVIEW

**Comment by h3poteto** [comment]
At first I have to add new language in crowdin configuration, so please wait.

File: src/config/locales/gd/translation.json [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+    "poll_vote": {
+      "title": "Bhòt cunntais-bheachd",
+      "body": "Cuir {{username}} bhòt ris a’ chunntas-bheachd agad"
+    },
+    "poll_expired": {
+      "title": "Crìoch cunntais-bheachd",
+      "body": "Thàinig an cunntas-bheachd aig {{username}} gu crìoch"
+    }
+  }
+}
```
Comment (h3poteto) [inline src/config/locales/gd/translation.json:542]: Don't edit `translation.json` file. This file will be automatically updated by crowdin. So please write words on crowdin.

Comment (gunchleoc) [reply src/config/locales/gd/translation.json:542]: OK, will do. Don't worry about merging this and having it overwritten, I can reconstruct everything in my translation software :)

Comment (gunchleoc) [reply src/config/locales/gd/translation.json:542]: Thank you very much and I apologize for the extra work. Crowdin let me upload the json file, so the translation is now complete over there :)

I have opened 2 source string issues at Crowdin.

**Pull request closed without merging**